src/ResourceManager/RecoveryServices.SiteRecovery/Commands.RecoveryServices.SiteRecovery/help/Update-AzureRmRecoveryServicesAsrProtectionDirection.md

```diff
@@ -14,19 +15,46 @@ Updates the replication direction for the specified replication protected item o
 ### ByRPIObject (Default)
 ```
 Update-AzureRmRecoveryServicesAsrProtectionDirection -ReplicationProtectedItem <ASRReplicationProtectedItem>
- -Direction <String> [-WhatIf] [-Confirm] [<CommonParameters>]
+ -Direction <String> [-DefaultProfile <IAzureContextContainer>] [-WhatIf] [-Confirm] [<CommonParameters>]
 ```
 
-### ByRPObject
+### AzureToVMware
 ```
-Update-AzureRmRecoveryServicesAsrProtectionDirection -RecoveryPlan <ASRRecoveryPlan> -Direction <String>
- [-WhatIf] [-Confirm] [<CommonParameters>]
+Update-AzureRmRecoveryServicesAsrProtectionDirection [-AzureToVMware] [-Account <ASRRunAsAccount>]
+ -DataStore <ASRDataStore> [-MasterTarget <ASRMasterTargetServer>] -ProcessServer <ASRProcessServer>
+ -ProtectionContainerMapping <ASRProtectionContainerMapping>
+ -ReplicationProtectedItem <ASRReplicationProtectedItem> -Direction <String>
+ -RetentionVolume <ASRRetentionVolume> [-DefaultProfile <IAzureContextContainer>] [-WhatIf] [-Confirm]
+ [<CommonParameters>]
 ```
 
-### ByPEObject
```

Comment:
@viverm there is no parameter set that the removed `ByPEObject` parameter set can now apply to, which is a breaking change

Reply:
there was no existing handling of this code Parameterset ByPEObject. 
@cormacpayne  We can do for parameter set PEObject  put an deprecation Message and remove it later releases.

    switch (this.ParameterSetName) 
                 { 
                     case ASRParameterSets.ByRPIObject: 
                         this.protectionContainerName = Utilities.GetValueFromArmId( 
                             this.ReplicationProtectedItem.ID, 
                             ARMResourceTypeConstants.ReplicationProtectionContainers); 
                         this.fabricName = Utilities.GetValueFromArmId( 
                             this.ReplicationProtectedItem.ID, 
                             ARMResourceTypeConstants.ReplicationFabrics); 
                         this.SetRPIReprotect(); 
                         break; 
                     case ASRParameterSets.ByRPObject: 
                         this.SetRPReprotect(); 
                         break; 
                 }